ApiDoctor.Validation/Http/HttpParser.cs

```diff
@@ -23,115 +23,118 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+
+
+using System.Globalization;
+using System.IO;
+using System.Web;
+
 namespace ApiDoctor.Validation.Http
 {
     using System;
-    using System.IO;
+    using System.Collections.Specialized;
+    using System.Linq;
     using System.Net;
-    using System.Collections.Generic;
-
     public class HttpParser
     {
-        
-        
         /// <summary>
-        /// Converts a raw HTTP request into an HttpWebRequest instance.
+        ///     Converts a raw HTTP request into an HttpWebRequest instance.
         /// </summary>
         /// <param name="requestString"></param>
         /// <returns></returns>
         public HttpRequest ParseHttpRequest(string requestString)
         {
             if (string.IsNullOrWhiteSpace(requestString))
-            {
                 throw new ArgumentException("requestString was empty or whitespace only. Not a valid HTTP request.");
-            }
 
-            StringReader reader = new StringReader(requestString);
+            var firstNewLine = requestString.IndexOf(Environment.NewLine, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
what's this trying to do? trim any newlines before the request starts?
1) why not just fix the docs? i think that's what i did for v1.
2) why not just call trim()?

Reply:
I realized I was wrong here. I fixed this.